### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by sebastiangaertner**
Please review. These are small changes mainly sorting out the spelling of Treetracker which should be one word across. The only line i swas not sure about is line 4.  this looks like a comment to me so please confirm. I will work on the formatting of the Legal Screen and the About screen as well. These are cosmetic but important changes.

### CODE REVIEW

**Comment by sebastiangaertner** [comment]
We should also format the policy_text_blob and the information_text both rendered as WebView

since this https://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling is very limiting.

Tutorial: https://alvinalexander.com/android/how-show-html-string-in-android-textview-webview

**Comment by sebastiangaertner** [comment]
I have also changed the Gradle file and in the app name changed the name from Tree Tracker to Treetracker. This should only effect the display of it not any package names.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -208,85 +208,35 @@
...
-
-It is our intent to hold the principles of this policy, but we do understand there may be a need to change it from time to time. Changes will be posted on our website as soon as possible.
-        \n
-        \n
-
-© 2019 GreenStand (registered 501(c)3 nonprofit. EIN: 47-3150283 ). Info@greenstand.org
-        \n
-Privacy Policy
-
+	    <![CDATA[
```
Comment (Elforama) [inline app/src/main/res/values/strings.xml]: Please undo this HTML change. This should be done in a different PR. This requires code changes.
